src/Microsoft.Health.Fhir.Shared.Web/DevelopmentIdentityProviderRegistrationExtensions.cs

```diff
@@ -188,6 +186,30 @@ private static IReadOnlyCollection<ApiScope> GenerateSmartClinicalScopes()
             return scopes;
         }
 
+        private static IEnumerable<ClientClaim> CreateFhirUserClaims(string userId)
+        {
+            string userType = null;
+
+            if (userId.Contains("patient", StringComparison.OrdinalIgnoreCase))
+            {
+                userType = "Patient";
+            }
+            else if (userId.Contains("practitioner", StringComparison.OrdinalIgnoreCase))
+            {
+                userType = "Practitioner";
+            }
+            else if (userId.Contains("system", StringComparison.OrdinalIgnoreCase))
+            {
+                userType = "System";
+            }
+
+            return new ClientClaim[]
+            {
+                new ClientClaim("appid", userId),
+                new ClientClaim("fhirUser", $"https://localhost:44348/{userType}/" + userId),
```

Comment:
Should we have "https://localhost:44348/" hard coded? 
I think there is a local configuration in applicationSettings.json.

Reply:
Fixed to use configuration value.